src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int BALL_COUNT = 100;
+
+    public Ball getRandomBall() {
+        Ball randomBall = new Ball();
+        randomBall.setColor(new ColorSupplier().getRandomColor());
+        randomBall.setNumber(new Random().nextInt(BALL_COUNT));
```

Comment:
it's bad practice to initialize dependencies in the method. you can initialize it in fields, 
 but good practice initialization using a constructor:
```
    private final Random random;
    private final ColorSupplier supplier;

    public Lottery(Random random, ColorSupplier supplier) {
        this.random = random;
        this.supplier = supplier;
    }
```